content/docs/installation.md

```diff
@@ -40,9 +40,9 @@ cd my-app
 npm start
 ```
 
-Create React App doesn't handle backend logic or databases; it just creates a frontend build pipeline, so you can use it with any backend you want. It uses build tools like Babel and webpack under the hood, but works with zero configuration.
+`Create React App` doesn't handle backend logic or databases; it just creates a frontend build pipeline, so you can use it with any backend you want. It uses build tools like [Babel] (http://babeljs.io/) and [webpack] (http://babeljs.io/) under the hood, but works with zero configuration.
```

Comment:
The webpack link is incorrect. Will be good to reomve the spaces as well as mentioned earlier.

Reply:
I don't think Create React App should be formatted as code. "Create React App" is a title, vs `create-react-app` which is the tool.